input/vlct/HD_linear_wave/method_vlct_hd_entropyN16.in

```diff
@@ -0,0 +1,14 @@
+# Problem: 3D LinearWave test problem for VLCT
```

Comment:
I agree, it's very cumbersome. To my knowledge, enzo-e currently only accepts the parameter file as command line argument.

It would be fantastic to come up with some kind of workaround.